src/app/medInria/main.cpp

```diff
@@ -274,10 +271,6 @@ int main(int argc,char* argv[])
 
     application.setMainWindow(mainwindow);
 
-#ifdef USE_PYTHON
-    med::python::startConsole();
```

Comment:
It is removed here but put elsewhere in https://github.com/medInria/medInria-public/pull/1000 (the two go together as described in that PR ). It will also change again in another PR I haven't posted yet. I have refactored medPython many times because of various bugs, and this affects the places where the Python components (interpreter, console, settings widget, plugins) are setup.